### ISSUE TITLE
Stock Data Analytics Stack

### ISSUE THREAD

**Issue by ThaliaBarrera** [airbytehq/quickstarts#63]
### Stock Market Analysis with Airbyte

Extract real-time or historical stock data, news, or financials from the Polygon Stock API using Airbyte. Load the data into a warehouse to perform trend analysis, predictions, or portfolio evaluations.

You can use a tool like dbt for data transformation, and an orchestrator like Airflow or Dagster if needed.

How to get started:

- Check out the [README](https://github.com/airbytehq/quickstarts/tree/main) and the [Contribution instructions](https://github.com/airbytehq/quickstarts/blob/main/CONTRIBUTING.md) in this repo.
- Check one of the [existing quickstarts](https://github.com/airbytehq/quickstarts#list-of-available-quickstarts) and observe their structure.
- (Optional, but highly recommended) Implement one of the quickstarts by following the instructions in their README. I recommend starting with [E-commerce analytics in BigQuery](https://github.com/airbytehq/quickstarts/tree/main/ecommerce_analytics_bigquery)

**Comment by Ronlin1** [comment]
Assign this to me please

**Comment by ThaliaBarrera** [comment]
Hi @Ronlin1! Sure, I have assigned it to you.

**Comment by ThaliaBarrera** [comment]
Hi @Ronlin1! Are you still working on this? Otherwise I may need to unassign. Let me know :)

**Comment by Ronlin1** [comment]
Yes, I am still on it: Finalising probably tomorrow.

**Comment by Ronlin1** [comment]
Can i use only Airbyte UI for this: and any database(mongodb) since I am facing some pc issues for airbyte deployment?

**Comment by ThaliaBarrera** [comment]
> Can i use only Airbyte UI for this: and any database(mongodb) since I am facing some pc issues for airbyte deployment?

@Ronlin1 Yes, you can use the Airbyte UI instead of Terraform. And sure, you could use Mongo as a source if you have it populated with data. I just thought it's easier to pull it from a public API, but feel free to use another source!
